api/app/controllers/api/v1/assignments_controller.rb

```diff
@@ -44,6 +49,17 @@ def update
             end
         end
 
+        # /assignments/delete or /assignments/:id/delete
+        def delete
+            params.require(:id)
+            assignment = Assignment.find(params[:id])
+            if session.destroy!
```

Comment:
Why are we destroying the session? Shouldn't we be destroying the assignment?

Reply:
Good catch. I'll fix this. Thanks.